operations/bills.md

```diff
@@ -184,12 +223,26 @@ A bill is either a `Receipt` which means that it has been fully paid, or `Invoic
 
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
-| `ItalianFiscalCode` | string  | optional | Italian fiscal code. |
+| `Id` | string  | optional | ID of the [Customer](customers.md#customer) to whom the bill was assigned. |
```

Comment:
is it correct to use "to whom the bill was assigned" here, or should it refer to the owner of the bill?

Reply:
At the moment, this refers to a customer to whom the bill was billed. In other words a customer's bill. It's not interchangeable with company.